www/templates/build.html

```diff
@@ -359,8 +359,17 @@ <h2 class="head-2">
             <div id="artifacts-js">
                 No artifacts
             </div>
-
-
+            <div class="divider"></div>
+            <div>
+                <a class="small-head" href={{ build_url+"?debug=true" }}>Show hidden steps</a>
+            </div>
+            {% if result_css == "failure" %}
+            <div>
+                <a class="small-head" target="_blank" href="https://q.unity3d.com/questions/244/how-do-i-connect-to-a-build-machine.html">
```

Comment:
could you add an entry in master.cfg that has the help url so we dont have it hard coded here ?